files/justfiles/toggles.just

```diff
@@ -177,22 +177,74 @@ toggle-xwayland ACTION="prompt":
 
 # Toggle bash environment lockdown (mitigates LD_PRELOAD attacks)
 toggle-bash-environment-lockdown:
-    #!/usr/bin/bash
-    BASH_ENV_FILES=("$HOME/.bashrc" "$HOME/.bash_profile")
+    #!/bin/run0 /bin/bash
+    set -eou pipefail
+    user_string=$(ls "/var/home" | tr '\n' ',' | sed 's/,*$//')
+    IFS=',' read -ra user_list <<< "$user_string"
+    [ "$UID" -eq 0 ] || { echo "This script must be run as root."; exit 1;} #almost certainly not needed, but doesn't hurt
+    declare -a filtered_list
+    for element in "${user_list[@]}"; do
+        if [ ! -d "/var/home/$element/Desktop/" ]; then
+                break
+        fi
+        filtered_list+=("$element")
+    done
+    b=$(tput bold) #This is the more correct way to bold and satisfies -eou pipefail
+    n=$(tput sgr0)
     echo "${b}WARNING${n} This will overwrite your .bashrc and .bash_profile."
     echo "This is needed to ensure the mitigation is effective."
     echo "Do you understand?"
     echo "Please type in \"YES I UNDERSTAND\" and press enter"
-    read ACCEPT
-    if [ "$ACCEPT" == "YES I UNDERSTAND" ]; then
-      if lsattr "${BASH_ENV_FILES[0]}" 2>/dev/null | awk '{print $1}' | grep -q 'i'; then
-        echo "Bash environment '(${BASH_ENV_FILES[@]})' is locked down. Unlocking it."
-        for file in "${BASH_ENV_FILES[@]}"; do
-            run0 chattr -i "$file"
+    read -r ACCEPT
+    if [ "$ACCEPT" != "YES I UNDERSTAND" ]; then
+        exit
+    fi
+    echo "Do you want this change to apply to ${b}all${n} users? [y/N]"
+    echo "Otherwise, it will only apply to the currently logged in user."
+    read -r REPLY
+    if [[ ! "$REPLY" == [yY]* ]]; then
+        filtered_list[0]=$SUDO_USER
+        filtered_list[1]=$SUDO_USER
+        if lsattr "/etc/profile" 2>/dev/null | awk '{print $1}' | grep -q 'i'; then
+                change="unlocked"
+        else
+                change="locked"
```

Comment:
`change` is unconditionally overridden by the identical looking block of code down below, which means that the entire block here can be simply removed.